perf_dashboard/clientlibs_download/python_clientlibs_download.py

```diff
@@ -14,11 +14,16 @@
 
 import datetime
 import os
+import sys
 import time
 import uuid
 
 from google.cloud import bigquery
 
+sys.path.insert(0, os.path.abspath(__file__+"/../../.."))
```

Comment:
Can you add a comment describing why we need this hack?

Reply:
Added.